backend/src/test/java/com/redhat/cloud/notifications/routers/sources/SecretUtilsTest.java

```diff
@@ -175,19 +193,24 @@ void createSecretsForEndpointBasicAuthNullTest() {
         final Secret secretTokenMock = new Secret();
         secretTokenMock.id = SECRET_TOKEN_SOURCES_ID;
 
+        // Set the ID for the secret bearer that is supposed that is created in Sources.
+        final Secret secretBearerMock = new Secret();
+        secretBearerMock.id = BEARER_TOKEN_SOURCES_ID;
+
         // Create an endpoint that contains the expected data by the function under test.
         final String orgId = "create-secrets-for-endpoint-basic-auth-null-test-organization-id";
 
         Endpoint endpoint = new Endpoint();
         WebhookProperties webhookProperties = new WebhookProperties();
         webhookProperties.setSecretToken(SECRET_TOKEN);
+        webhookProperties.setBearerAuthentication(BEARER_TOKEN);
 
         endpoint.setProperties(webhookProperties);
         endpoint.setOrgId(orgId);
 
         // Set up the mock call for the "create" call from the REST Client. In this case, since the basic
         // authentication is null, only the secret token should be created.
```

Comment:
```diff
-      // Set up the mock call for the "create" call from the REST Client. In this case, since the basic
-      // authentication is null, only the secret token should be created.
+        // Set up the mock call for the "create" call from the REST Client. In
+        // this case, since the basic authentication is null, only the secret
+        // token should be created.